packages/core/src/__integrationtests__/utils.ts

```diff
@@ -67,3 +90,111 @@ export const IsOfficialLicenseAuthority = CType.fromSchema({
   },
   type: 'object',
 })
+
+export async function initialTransfer(
```

Comment:
W: Would rename to something like `endowAccounts`?

Reply:
Updated.